finetuning/specialists/training/light_microscopy/common.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+from math import ceil, floor
+
+from torch_em.transform.label import label_consecutive
+from torch_em.transform.label import PerObjectDistanceTransform
+from torch_em.transform.raw import normalize_percentile, normalize
+
+
+class ResizeRawTrafo:
```

Comment:
Let's not duplicate this functionality over and over. It's better if you move this to `micro_sam.training.util`.
Same is true for `ResizeLabelTransform`. And you should also do this for the code in https://github.com/computational-cell-analytics/micro-sam/pull/295.

Reply:
That's a good point. I will refactor these later then (once the joint training bug is taken care of). Thanks!